wordpress-mobile/WordPress-iOS
# Update News Card for Stats.

Fixes #n/a

This updates the News card to show info about Stats.

To test:
- Go to the Reader.
- Verify the News card now references Stats.
- Verify `Read more` opens the Stats en.blog post. 

![news_card](https://user-images.githubusercontent.com/1816888/62244084-eaacb280-b39b-11e9-88d8-b93647485cbf.png)

Update release notes:
